buidler.config.js

```diff
@@ -6,24 +6,8 @@ module.exports = {
   defaultNetwork: 'localhost',
   networks: {
     localhost: {
-      url: 'http://localhost:8545',
-      accounts: [
-        '0xa8a54b2d8197bc0b19bb8a084031be71835580a01e70a45a13babd16c9bc1563',
-        '0xce8e3bda3b44269c147747a373646393b1504bfcbb73fc9564f5d753d8116608'
-      ]
+      url: 'http://localhost:8545'
     },
-    buidlerevm: {
-      accounts: [
-        {
-          privateKey: '0xa8a54b2d8197bc0b19bb8a084031be71835580a01e70a45a13babd16c9bc1563',
-          balance: '100000000000000000000'
-        },
-        {
-          privateKey: '0xce8e3bda3b44269c147747a373646393b1504bfcbb73fc9564f5d753d8116608',
-          balance: '100000000000000000000'
-        }
-      ]
-    }
   },
   solc: {
     version: '0.4.24'
```

Comment:
Do we want to have a pre configured amount of runs here?

Reply:
Hmmm. Yeah, I'll add that in.